src/coreclr/nativeaot/BuildIntegration/WindowsAPIs.txt

```diff
@@ -1263,7 +1263,6 @@ kernel32!LCMapStringEx
 kernel32!LCMapStringW
 kernel32!LeaveCriticalSection
 kernel32!LeaveCriticalSectionWhenCallbackReturns
-kernel32!LoadLibraryA
```

Comment:
Why remove it from here? P/Invokes to `LoadLibraryA` in NativeAOT will no longer be emitted as direct calls. Not sure if this is what you want.

Reply:
Oops. This is wrong. Thanks.